Client/Frontend/Browser/URIFixup.swift

```diff
@@ -6,9 +6,13 @@ import Foundation
 
 class URIFixup {
     static func getURL(entry: String) -> NSURL? {
-        let trimmed = entry.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
+        let allowedCharacters = NSMutableCharacterSet(charactersInString: "%")
+        allowedCharacters.formUnionWithCharacterSet(NSCharacterSet.URLFragmentAllowedCharacterSet())
+        guard let trimmed = entry.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet()).stringByAddingPercentEncodingWithAllowedCharacters(allowedCharacters) else {
```

Comment:
I think we'll want a separate variable for the percent escapes, using the escaped version in this first check, but using `trimmed` in the last check before the `return`. 

The reason: the last check prepends `http://` before the string, so loosening up the URL requirements might make us use URLs for queries that aren't actually URLs. Example: if I search `dr. who`, the escaped string will be `dr.%20who`, and with the prepended `http://`, it becomes `http://dr.%20who` -- which would probably be interpreted as valid URL rather than a search.